src/main/java/com/example/springresttask/domain/Email.java

```diff
@@ -0,0 +1,31 @@
+package com.example.springresttask.domain;
+
+import java.time.LocalDateTime;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
```

Comment:
You can replace multiple annotations here with: `lombok.Data`.
So it will take less code to have the same piece of functionality.